core/src/main/java/com/novoda/merlin/MerlinsBeard.java

```diff
@@ -14,9 +21,13 @@
 public class MerlinsBeard {
 
     private static final boolean IS_NOT_CONNECTED_TO_NETWORK_TYPE = false;
+    private static final String ENDPOINT = "http://clients3.google.com/generate_204";
```

Comment:
should this be `https`? is this working on Android 9?

Reply:
instead of naming this `ENDPOINT`, shouldn't be `CAPTIVE_PORTAL_ENDPOINT` or something alike?

Reply:
No, it's used for all of our endpoint calls now.

Reply:
It should in theory work on android 9.

Reply:
Need to remove this, we are using `default` which is essentially the same, it's an endpoint that is guaranteed to return a `204`.

Reply:
Can't we use `DEFAULT_ENDPOINT` from `Endpoint`?